.github/workflows/issue-triage.yml

```diff
@@ -13,3 +13,11 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           enabled: true
+  add-to-project:
+  name: Add issue to Design System project
+  runs-on: ubuntu-latest
+  steps:
+    - uses: actions/add-to-project@v0.0.3
+      with:
+        project-url: https://github.com/orgs/carbon-design-system/projects/39
+        github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
Would this section need to be indented for the options to be set on the `add-to-project` job?

Reply:
@joshblack I don't think so? [Docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-6) have it in line with `uses`, `with` doesn't have the dash though.

```yml
jobs:
  my_first_job:
    steps:
      - name: My first step
        uses: actions/hello_world@main
        with:
          first_name: Mona
          middle_name: The
          last_name: Octocat
```

Unless I'm misunderstanding what you're asking?

Reply:
Fixed!